MonoGame.Framework.Content.Pipeline/Builder/PathHelper.cs

```diff
@@ -65,7 +65,7 @@ public static string NormalizeOS(string path, TargetPlatform targetPlatform)
             switch (targetPlatform)
             {
                 case TargetPlatform.Windows:
-                case TargetPlatform.WindowsGL:
+                case TargetPlatform.DesktopGL:
```

Comment:
I's rather not alter the current behavior with a PR that's unrelated to PathHelper.